lib/feature_management.rb

```diff
@@ -48,8 +48,9 @@ def self.enable_identity_verification?
     Figaro.env.enable_identity_verification == 'true'
   end
 
-  def self.reveal_usps_code?
-    Rails.env.development? || current_env_allowed_to_see_usps_code?
+  def self.reveal_usps_flow?
+    env = Rails.env
+    env.development? || env.test? || current_env_allowed_to_see_usps_code?
```

Comment:
What do you think about tying this feature flag to a Figaro entry, similarly to our `enable_identity_verification` flag so that we can easily turn it on when we're ready without having to modify the IdP code?

Also, did you mean to remove the `reveal_usps_code?` flag? IIUC, they serve 2 separate purposes. One is to facilitate testing of the USPS flow feature by prefilling the code in the form, and should only be enabled in certain environments. The other is to globally turn on or off the whole USPS feature. If the USPS feature is ON, we still want the code prefilling to be OFF on some servers. In this PR, if I'm reading things correctly, turning on the USPS feature will automatically also prefill the USPS code.